src/devices/Mcp25xxx/Mcp25xxx.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
```

Comment:
use `$(SystemDeviceGpioPackageVersion)` instead of the specific version (recently changed everywhere)

Reply:
Will update.

Reply:
I'm going to hold off on this until the end.  There are some other changes in the repo that have since been changed and break my local build when doing this.